common/copr_common/dispatcher.py

```diff
@@ -126,3 +129,20 @@ def run(self):
 
         # reset the title
         self._update_process_title()
+
+    def dump_queue(self, worker_manager):
+        """
+        Dump the current priority queue state into a JSON file
```

Comment:
Can you put in the docstring the reason why we are doing this? So your future yourself do not worry it is dead code.

Reply:
It is not a dead code actually ... it is dumped in every single iteration, which raises my worries.  Could we have an opt-in knob, and document there when we need it?